sct-commons/src/main/java/org/lfenergy/compas/sct/commons/dto/LDEPFSettings.java

```diff
@@ -0,0 +1,24 @@
+// SPDX-FileCopyrightText: 2023 RTE FRANCE
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.lfenergy.compas.sct.commons.dto;
+
+import org.lfenergy.compas.sct.commons.util.SettingLDEPFCsvHelper;
```

Comment:
Interface must not have a dependence on its implementation.

Reply:
OK, I set no type restrictions